frontend/src/assets/css/scaffolding.scss

```diff
@@ -40,7 +38,7 @@ body {
     padding: 0 var(--p-3);
 }
 
-@media (max-width: $md) {
+@media (max-width: 768px) {
     body {
         padding-bottom: 50px;
     }
```

Comment:
Doesn't work otherwise

Reply:
Did you try to use `@import "@assets/utils";`